rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -32,6 +32,8 @@ using namespace std;
 void COCOMetaDataReader::init(const MetaDataConfig &cfg, pMetaDataBatch meta_data_batch)
 {
     _path = cfg.path();
+    _avoid_class_remapping = cfg.class_remapping();
```

Comment:
Class remapping we need not set to this pointer?
Have you tested and validated these?

Reply:
_avoid_class_remapping is a variable in the metadata reader whereas the aspect ratio grouping is set in the metadata reader using this function

Reply:
Why is this difference followed for the 2 variables ?

Reply:
avoid_class_remapping is used within this class itself so its accessible but we need aspect_ratio_grouping variable accessible in coco_file_source_reader. Private variables won't be accessible outside class so added a public function to get the value and use it